library/network/src/modules/SuSEFirewall.rb

```diff
@@ -67,6 +67,23 @@ def self.get_installed_backends
       backends
     end
 
+    # Obtain list of enabled backends.
+    #
+    # @return array of symbols representing enabled backends.
+    def self.get_enabled_backends
+
+      Yast.import "Service"
+
+      enabled_backends = []
+
+      installed_backends = self.get_installed_backends
+      installed_backends.each do |b|
+        enabled_backends << b if Service.Enabled(@@firewall_backends[b])
```

Comment:
I actually think that more convenient method alias is `select` before `find_all`